modules/ROOT/partials/konflux-key-features.adoc

```diff
@@ -1,27 +1,52 @@
+
+* *Hermetic Builds:* {ProductName} isolates your builds from changes to the host system to ensure consistency and produce accurate Software Bills of Materials (SBOMs) for tracking components, dependencies, and so forth.
+
+* *SLSA Compliance*: {ProductName} supports level three Supply Chain Levels for Software Artifacts (SLSAs), allowing you to generate secure, detailed records for each build step to ensure transparency and trust in your software supply chain.
 
 +
-Safeguard sensitive data with {ProductName}'s stringent access controls and governance policies, ensuring rigorous data protection.
+*Note*: link:https://slsa.dev[SLSA] is a security framework created through industry collaboration. We use this framework as a guide to reinforce the build process we use for your applications, thereby enhancing the security of your software supply chain. To see how we meet the requirements set out by SLSA, see xref:metadata:index.adoc#supply-chain-security-through-slsa-conformity[Supply chain security through SLSA conformity].
+
+* *Stringent Access Controls*: {ProductName} allows you to implement policies to safeguard your sensitive data and ensure access compliance without giving up flexibility.
+
+* *Vulnerability Detection*: {ProductName} automatically scans for critical vulnerabilities every time you submit a pull request and verifies your container images against custom rules and secure frameworks.
+
+=== Strict Data Governance
+
+{ProductName} safeguards your sensitive data with stringent access controls and governance policies, ensuring rigorous data protection.
+
+=== Testing and Deployment
+
+{ProductName} provides the following features to facilitate testing and deploying your software:
 
-*Choose {ProductName} — where efficiency meets security.*
+* *Build Once, Deploy Anywhere*: {ProductName} optimizes your release process for multiple deployments without redundancy, which reduces your time-to-market
 
-== Supply chain security through SLSA conformity
+* *Integration Testing*: {ProductName} runs your integration tests for complex applications and displays the results in your source control management system.
 
-link:https://slsa.dev[Supply-chain Levels for Software Artifacts (SLSA)] is a security framework produced through industry collaboration. We use this framework as a guide for reinforcing the build process we use for your applications, to better secure your software supply chain.
+* *Supports Multiple Architectures*: {ProductName} supports architectures like *amd64* and *arm64*, offering you flexibility when targeting diverse deployment environments.
 
-To see how we meet the requirements set out by SLSA, see xref:metadata:index.adoc#supply-chain-security-through-slsa-conformity[Supply chain security through SLSA conformity]
+* *OpenShift Clusters*: {ProductName} provides OpenShift clusters, enabling teams to test applications, including those that require administrative access for Operator Lifecycle Manager (OLM) operators.
```

Comment:
I don't understand meaning of this, how konflux provides Openshift Clusters?

Reply:
Oh, this must be through environment-as-a-service (eaas) or cluster-as-a-service (caas) bits: https://developers.redhat.com/articles/2024/10/28/ephemeral-openshift-clusters-konflux-ci-using-cluster-service-operator

@amisstea, from your PoV does it make sense to mention or advertise that in the upstream docs? If so, we should classify it as a "konflux add-on". What name should be used to refer to it upstream?

Reply:
I don't think we should include this in the upstream docs. We've only ever deployed or tested EaaS as a downstream add-on.